portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,24 +32,26 @@
 import java.util.*;
 
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns user comments. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     // Private class to represent comments as objects in Datastore.
     private class Comment {
         private long id;
         private String text;
+        private long time;
```

Comment:
time_millis ? Makes it clear what the time value should be.

Reply:
Changed to `timestamp` and switched to a more descriptive, human-readable format (e.g.  Jun 5, 2020 9:55:05 PM). Might be more precise than needed.